File: Java/src/Faccat/Exercicio6a.java
package Faccat;

import java.util.Scanner;

public class Exercicio6a {
    public static void main(String[] args) {
        double base, altura, area;
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o valor da base: ");
        base = scanner.nextDouble();

        System.out.print("Digite o valor da altura: ");
        altura = scanner.nextDouble();

        area = altura * base;

        System.out.println("A área é: " + area);

        scanner.close();
    }
}

File: JavaScript/Faccat/Exercicio6a.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Digite o valor da base: ', function(base) {
  rl.question('Digite o valor da altura: ', function(altura) {
    const area = parseFloat(base) * parseFloat(altura);
    console.log('A área é: ' + area);

    rl.close();
  });
});